### ISSUE TITLE
Pagination NOLIMIT is not applied

### ISSUE THREAD

**Issue by colonder** [Webbist-dev/strapi-redirects#22]
I used to get all the redirects by simply sending a request to `/api/redirects?pagination[start]=0&pagination[limit]=-1` but now it seems that the `pageSize` of 10 is applied by default. Adding `pagination[limit]=-1` doesn't do anything. Currently, one can obtain all of the redirects only by increasing `pageSize` to some high number like `999`.  I can see [in the code](https://github.com/Webbist-dev/strapi-redirects/blob/605bb00e9b9b078a74c828c159dcf47bbe0f752a/server/services/redirects.js#L4) that `NOLIMI` is not used anywhere.

**Comment by BL-Alex** [comment]
I'll look into this and get a fix for you.

**Comment by Webbist-dev** [comment]
Hey there, I've cleaned up the code - noLimit isnt needed, I originally had a findAll route to get all, but the new findMany route handles pagination as default in the rest response. 

Try this on your url. 

`/api/redirects?pagination[pageSize]=-1`

Check Strapi docs for pagination, I've mapped it to work off the same principles. 

[Strapi Pagination docs](https://docs.strapi.io/dev-docs/api/rest/sort-pagination)
